test/test_cpp_models.py

```diff
@@ -25,15 +25,17 @@ def process_model(model, tensor, func, name):
 
 
 def read_image1():
-    image_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'assets', 'grace_hopper_517x606.jpg')
+    image_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'assets', 'encode_jpeg',
+                              'grace_hopper_517x606.jpg')
```

Comment:
I had to move the files related to jpeg decoding/writing to their own folder. Some of the tests on `test_image.py` start writing in random locations if they spot a jpeg and this interferes with the changes of this PR. Unfortunately the `grace_hopper_517x606.jpg` is used by a few other tests so here I have to update the locations. See below, my remarks on the problematic tests.

Reply:
Created issue #3002